### PR TITLE
added onBlur and onFocus props

### PR DESCRIPTION
**Pull request by erichhsun**
Updated version of pull request #79

### CODE REVIEW

**Comment by hanford** [comment]
+1

**Comment by lookapanda** [comment]
+1

**Comment by rogeriochaves** [comment]
+1 please merge this

**Comment by rogeriochaves** [comment]
I've solved conflicts on #167

File: lib/Autocomplete.js [hunk 1]
```diff
@@ -332,8 +338,8 @@ let Autocomplete = React.createClass({
           aria-autocomplete="both"
           autoComplete="off"
           ref="input"
-          onFocus={this.handleInputFocus}
-          onBlur={this.handleInputBlur}
+          onFocus={(event) => this.handleInputFocus(event)}
+          onBlur={(event) => this.handleInputBlur(event)}
```
Comment (Seldaek) [inline lib/Autocomplete.js]: This is not necessary, you just create an additional function and extra indirection for nothing. onFocus={callback} will call callback with the event already, much like it calls this inline function with the event.

**Pull request closed without merging**